### ISSUE TITLE
Object.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by fabriziogiordano** [jchouse/csscomb-atom#11]
Requiring `EditorView` from `atom` is no longer supported.
Please require `TextEditorView` from `atom-space-pen-view` instead:
  `{TextEditorView} = require 'atom-space-pen-views'`
Add `"atom-space-pen-views": "^2.0.3"` to your package dependencies.
```
Object.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/exports/atom.js:68:11)
Object.<anonymous> (/Users/giordanof/.atom/packages/atom-css-comb/lib/css-comb-view.coffee:1:7)
```

**Comment by jchouse** [comment]
Fixed in v0.6.5. Will be available in the next release

**Comment by jchouse** [comment]
Fixed in v0.6.5. Will be available in the next release
